transfer/adapterbase.go

```diff
@@ -124,11 +124,18 @@ func (a *adapterBase) worker(workerNum int, ctx interface{}) {
 		}
 		tracerx.Printf("xfer: adapter %q worker %d processing job for %q", a.Name(), workerNum, t.Object.Oid)
 
+		// tt is the time that we are to compare the transfer's
+		// `expired_at` property against.
+		tt := time.Now().Add(-objectExpirationGracePeriod)
+
 		// Actual transfer happens here
 		var err error
-		if t.Object.IsExpired(time.Now().Add(objectExpirationGracePeriod)) {
+		if expAt, expired := t.Object.IsExpired(tt); expired {
 			tracerx.Printf("xfer: adapter %q worker %d found job for %q expired, retrying...", a.Name(), workerNum, t.Object.Oid)
-			err = errors.NewRetriableError(errors.Errorf("lfs/transfer: object %q has expired", t.Object.Oid))
+			err = errors.NewRetriableError(errors.Errorf(
+				"lfs/transfer: object %q has expired at %s, %s ago",
```

Comment:
:+1: The time is also too precise, it doesn't need to show sub second values. Also, it should be in local time if possible, so the tz doesn't have to be set. Hopefully doing that conversion isn't too terrible :)

```diff
-has expired at 2016-10-21 14:24:33.728774785 -0600 MDT
+expires at 2016-10-21 14:24:33
```